hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/parsing/V2MessageUtil.java

```diff
@@ -167,6 +171,21 @@ public static String getMsgId(String v2Message) {
 	
 		return msgId;
 	}
+	
+
+	public static String getMsgCnrtlId(String v2Msg) {
```

Comment:
The shorthand for this method is incorrect It should be Cntrl not Cnrtl. Also, I believe Ctrl is a more standard abbreviation of Control...though I'm not sure the abbreviation of Control is necessary.